src/main/scala/com/cognite/sdk/scala/v1/transformations.scala

```diff
@@ -78,10 +78,12 @@ sealed case class RawDataSource(`type`: String, database: String, table: String)
 sealed case class SequenceRowDataSource(`type`: String, externalId: String)
     extends DestinationDataSource
 
+final case class TransformationRun(externalId: String)
+
 final case class JobDetails(
-    id: Int,
+    id: Long,
```

Comment:
this is int in jetfire 🤔

Reply:
not important but fyi

Reply:
I think it should be Long ? Maybe it jetfire we should change it to Long as well since the serialId can grow bigger than the 2147483647?

Reply:
not sure if it is necessary, we truncate the old jobs anyway. it will not grow that much and people don't create 1000 transformations everyday as in events or assets

Reply:
we can make it long in the api, but it is currently int.

Reply:
> not sure if it is necessary, we truncate the old jobs anyway. it will not grow that much and people don't create 1000 transformations everyday as in events or assets
I think the serialId keeps incremented anyway, it won't reuse the delete one until it hit the limit I think. I think it's fine we keep Long for now, better support larger type than smaller hehe